dotnet-desktop-guide/net/winforms/controls/how-to-set-the-tab-order.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Set tab order of controls
+description: Learn how to set the tab order of controls on your Windows Forms for .NET. Set the tab order with Visual Studio or using the TabIndex property in the Properties window.
+ms.date: 05/24/2021
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords:
+  - "tab order [Windows Forms], controls on Windows forms"
+  - "Windows Forms controls, setting tab order"
+  - "controls [Windows Forms], setting tab order"
+  - "Windows Forms, setting tab order"
+---
+# How to set the tab order on Windows Forms (Windows Forms .NET)
+
+The tab order is the order in which a user moves focus from one control to another by pressing the <kbd>Tab</kbd> key. Each form has its own tab order. By default, the tab order is the same as the order in which you created the controls. Tab-order numbering begins with zero and ascends in value, and is set with the <xref:System.Windows.Forms.Control.TabIndex%2A> property.
+
+You can also set the text by using the [designer](#designer).
```

Comment:
```suggestion
You can also set the tab order by using the [designer](#designer).

This sentence seems to fit better with the next paragraph, rather than being separated from it by an alert box.
```